### PR TITLE
Remove deprecated accounts

### PR DESCRIPTION
**Pull request by opoto**
As discussed in #897, removed some accounts:
- Digifit
- FunBeat
- Google Plus (moved OAUTH2 code to GoogleFit)
- Nike plus
- RunningFreeOnline

As suggested, kept RunnerUp LIVE in case someone reinstalls the server.
Did not dare remove Garmin, Facebook, Runtastic, MapMyRun, GoogleFit, and jogg.se, because these services still exist and someone may fix the synchronizers to bring them back to life.

### CODE REVIEW

**Comment by opoto** [comment]
You're right, I missed that.
What about deleting DB entries on the fly during account listing, when seeing that the synchronizer does not exist anymore?

**Comment by gerhardol** [comment]
> What about deleting DB entries on the fly during account listing, when seeing that the synchronizer does not exist anymore?

That is a variant of the current "always run" that disables accounts.
This will clear the status about uploaded activities.
Can you test how this behaves?

A variant is to set the account status to the new status "removed" at strtup and avoid displaying them in the account list.

It is hard to make such changes on your own, I have struggled for several years now without a discussion partner...

**Comment by gerhardol** [comment]
Follow up on RunningAHEAD: the service is flaky, but not dead. Let us leave it as is

**Comment by opoto** [comment]
I fully understand the difficulty to maintain this project alone, and I congratulate you for the work you do!
The on-the-fly approach I imagined seems to be too complex: there are many (complex) execution paths to read the accounts, and I'm sure it would miss a scenario. So DB upgrade seems to be the more reasonable way to remove these accounts.

**Comment by gerhardol** [comment]
A new db version adds a complexity when developing, I prefer to not do that.
Also, dbversion is not strictly required here as the format is not changed.

I was going to suggest to set ENABLED=2 in dbhelper, and add ENABLED!=2 to AccountListActivity.java, ManageWorkoutsActivity.java,
but deleting in the same way could be enough.

**Comment by opoto** [comment]
Deletion on the fly seems to work, I tested though 3 flows:
- open activity in history (although my activities did not refer to deleted accounts)
- feed
- accounts in preferences

**Comment by gerhardol** [comment]
Seem OK, will try tomorrow

File: app/src/main/play/listings/en-US/full-description.txt [hunk 1]
```diff
@@ -7,20 +7,19 @@ Track your sport activities with RunnerUp using the GPS in your Android phone:
...
 * Pebble support
 
-Upload your activities to a variety of running sites with a single click:
+Upload your activities to a variety of external applications with a single click:
 
 * Strava
 * Runalyze
 * Endomondo
-* RunKeeper
-* RunningAHEAD
```
Comment (gerhardol) [inline app/src/main/play/listings/en-US/full-description.txt:23]: Not dead yet, so keep, just less promoted

Comment (opoto) [reply app/src/main/play/listings/en-US/full-description.txt]: done

**Comment by gerhardol** [comment]
The TripleX doc (basis for F_Droid and Play) should be synced. Planned to do this myself

**Comment by opoto** [comment]
I don't know what is the TripleX doc, and did not find anything with that name in the repo...

**Comment by gerhardol** [comment]
app/src/main/play

Triple-T...
https://github.com/Triple-T/gradle-play-publisher

**Comment by opoto** [comment]
Updated and squashed

**Comment by gerhardol** [comment]
Thanks!

**Pull request merged**